.github/workflows/companion-deploy.yml

```diff
@@ -3,7 +3,7 @@ name: Companion Deploy
 on:
   push:
     tags:
-      - '@uppy/companion@*'
+      - '@uppy/companion@**'
```

Comment:
what's the purpose of this? [as far as i can see](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet), `**` is the same as `*` but it also matches slashes. Does companion version ever include a slash?

Reply:
Yeah that seems to be correct, but then why wasn't it triggered by the 4.0.1 release?

Reply:
Also not sure what's going on here then

Reply:
maybe something about how the tag was pushed? i think i experienced before that if you push the commit first without the tag, then separately push the tag later, then it doesn't trigger a build.

Reply:
or github actions doesn't run on commits from github actions? Perhaps to prevent loops?

Reply:
Ah yes, that's it!

Reply:
We would need to make the `release` workflow call this one, that should work.

Reply:
We could either look into https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run, or simply merge this workflow in the release one.